src/platform/forms-system/src/js/review/ObjectField.jsx

```diff
@@ -141,8 +141,14 @@ class ObjectField extends React.Component {
       if (!formContext.hideTitle && typeof title === 'function') {
         title = title(formData, formContext);
       }
+      const uiOptions = uiSchema['ui:options'] || {};
+      const ariaLabel = uiOptions.itemAriaLabel;
+      const itemName =
+        (typeof ariaLabel === 'function' && ariaLabel(formData || {})) ||
+        formData[uiOptions.itemName];
       const editLabel =
-        _.get('ui:options.ariaLabelForEditButtonOnReview', uiSchema) ||
+        (itemName && `Edit ${itemName}`) ||
+        uiOptions.ariaLabelForEditButtonOnReview ||
         `Edit ${title}`;
```

Comment:
But here, `itemName` can be null, which means we default to `ariaLabelForEditButtonOnReview` or the `title`. Should the label pattern here match what is happening in `ArrayField`?

Reply:
Ahhh good catch, thanks! I'll add this.

Reply:
Hmm, actually when I added a fallback to `'Item'`, it ends up that the `editLabel` always defaults to `Edit Item` because most object fields (not array fields) use the `title` on the review & submit page.

I added a duplicate `itemAriaLabel` to the ui options because the Military History section, new condition followup questions and VA treatment center name is rendered by the `ObjectField` on the review & submit page. Without the `itemAriaLabel` it would render a duplicate `'Edit {title}` for every entry.

![Screen Shot 2021-02-22 at 2 07 01 PM](https://user-images.githubusercontent.com/136959/108763773-5f63b680-7517-11eb-9e80-8bf88a71b5ab.png)

Reply:
I'm not sure what I'm supposed to be seeing in the screenshot. Is this a problem or is this ideal?

Reply:
We want the `aria-label` of the edit button to let a screenreader user know what the edit button will do. But as Micah pointed out, if the Veteran has more than one service period within the same branch, maybe we need to add a start date as well